pkg/health/server/server.go

```diff
@@ -341,11 +341,15 @@ func (s *Server) runActiveServices() error {
 		// Update set of nodes to probe every ProbeInterval and then fetch
 		// results
 		if nodesAdded, nodesRemoved, err := s.getNodes(); err != nil {
+			// reset the cache by setting clientID to 0 and removing all current nodes
+			s.clientID = 0
+			prober.setNodes(nil, prober.nodes)
```

Comment:
Should a probe retry interval with a more aggressive time than the probe interval be introduced to the server? This will allow the server to retry faster than waiting for the probe interval when an error is observed.

Reply:
Thanks for taking a look @danehans! I think since this is non-critical data we can just wait for the probe interval. The goal here was just to course-correct to make sure we're not displaying inaccurate data